resources/assets/less/bem/user-card-brick.less

```diff
@@ -4,18 +4,17 @@
 .user-card-brick {
   @_top: user-card-brick;
   @_fade-duration: 220ms;
-
-  @_background-color: @osu-colour-b6;
   @_border-radius: @border-radius-large;
 
   --border: transparent;
```

Comment:
not exactly this but `::after` should be combined with main block as it was only needed because of the background layers.

Additionally, it can be switched to use plain border and adjust the padding.